helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -305,6 +305,7 @@ internal fun OrtResult.getLicenseFindingsById(
             scanResult.summary.licenseFindings.let {
                 if (applyCurations) {
                     FindingCurationMatcher().applyAll(it, getLicenseFindingsCurations(scanResult.provenance))
+                        .mapNotNull { result -> result.curatedFinding }.distinct()
```

Comment:
Nit: I'd rather give the outer `it` the name `findings` to avoid name shadowing and use the implicitly `it` here.

Reply:
Done.